htsget_server/beacon_openapi.yaml

```diff
@@ -202,6 +203,12 @@ components:
         type: array
         items:
           type: string
+    fullSearch:
+      in: query
+      name: fullSearch
+      description: Set to true if a full beacon search is desired.
```

Comment:
can we add something to this description (and the one on line 720) about what you get with fullSearch=false? maybe "Set to true if a full beacon search is desired; default is to return estimated variant count"?

Reply:
```suggestion
      description: Set to true if a full beacon search is desired; default is to return estimated variant count.
```